src/components/tables/TransactionLogTable.less

```diff
@@ -0,0 +1,12 @@
+.transaction-log-table__status-bar {
+  font-size: 16px;
+  text-align: left;
```

Comment:
This is only applying to `Succeeded` states:
<img width="434" alt="screen shot 2018-08-06 at 10 27 21 am" src="https://user-images.githubusercontent.com/25164486/43725892-56904f9c-9963-11e8-9ff5-5f37ed9a1d19.png">